client/components/text/index.js

```diff
@@ -1,11 +1,34 @@
 import React, { PropTypes } from 'react';
 import FormFieldset from 'components/forms/form-fieldset';
+import FormLegend from 'components/forms/form-legend';
+import { sanitize } from 'dompurify';
+
+const renderTitle = ( title ) => {
+	if ( ! title ) {
+		return null;
+	}
+
+	return (
+		<FormLegend>{ title }</FormLegend>
+	);
+};
+
+const renderText = ( text ) => {
+	/* for dangerouslySetInnerHTML we need to pause linting for a narrow scope */
+	/* eslint-disable */
+	return (
+		<span dangerouslySetInnerHTML={ { __html: sanitize( text, { ADD_ATTR: ['target'] } ) } }>
```

Comment:
We don't presently - this is in case we ever want to be able to use this component to pass up an anchor with a target attribute.